lob/resource.py

```diff
@@ -170,6 +171,19 @@ def create(cls, **params):
         params.pop('objects', None)
         return super(Job, cls).create(**params)
 
+class Letter(ListableAPIResource, CreateableAPIResource):
+    endpoint = '/letters'
+    @classmethod
+    def create(cls, **params):
+        if isinstance(params, dict):
+            if 'from_address' in params:
+                params['from'] = params['from_address']
```

Comment:
from is a reserved python keyword so it complains if you try to use it as a param name in the create()